autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -56,7 +58,8 @@ def load(output_directory, verbosity=2):
         return TabularPredictor(learner=learner)
 
     @staticmethod
-    def fit(train_data, label, tuning_data=None, output_directory=None, problem_type=None, eval_metric=None, stopping_metric=None,
+    @unpack(set_presets)
```

Comment:
I think this messed up documentation for TabularPrediction.fit() on website

Reply:
Do you have any idea how to fix the documentation? The @unpack decorator is required for the presets functionality (it is actually impossible to do in Python without the decorator)

Reply:
This may be due to an unclosed ` in the docs, I will see if it fixes after the build finishes

Reply:
Nothing's jumping out at me looking through the fit docs for format-errors re what could be causing the error.  You may try moving the `@unpack` statement after `def fit()` and see if that allows the fit() doc to render, perhaps it breaks at `@unpack` but that's still ok if it's the final TabularPrediction function.  But I agree the bug is more likely some subtle sphinx violation in all the newly text of fit() doc.